src/App.jsx

```diff
@@ -48,6 +49,11 @@ function KanbanBoard() {
     
     if (!destination) return;
 
+    if (!columns[source.droppableId] || !columns[destination.droppableId]) {
+      console.error('Colunas não encontradas');
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const errorMessage = 'Columns not found';
console.error(errorMessage);
notifyError(errorMessage); // Implement proper error notification system
```
The error message for column operations is in Portuguese and lacks proper error handling structure. Consider implementing a proper error handling system with translated messages and error logging.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;